eng/dockerfile-templates/Dockerfile.linux.download-file

```
@@ -7,7 +7,10 @@
         sha: Expected checksum of the downloaded file
         sha-var-name: Name of variable that stores the checksum ^
 
+    set isInternal to find(ARGS["url"], "content?format=file&subPath") >= 0 ^
+    set additionalWgetArgs to when(isInternal, '--header="Authorization: Basic `echo $ACCESSTOKEN:$ACCESSTOKEN | base64 -w 0`" ', '') ^
```

Can't this be empty for the user name like is done for curl?

```suggestion
    set additionalWgetArgs to when(isInternal, '--header="Authorization: Basic `echo :$ACCESSTOKEN | base64 -w 0`" ', '') ^
```

---

Yeah, I wanted to do the same, but it doesn't work. It does work in curl and in Powershell (`Invoke-WebRequest` header).